.github/workflows/build.yml

```diff
@@ -0,0 +1,76 @@
+name: Build
+on:
+    push:
+        branches:
+            - master
+    pull_request:
+        branches:
+            - master
+jobs:
+    build:
+        name: PHP ${{ matrix.php-versions }} ${{ matrix.composer-args }}
+        strategy:
+            fail-fast: false
+            matrix:
+                operating-system: [ubuntu-latest]
+                php-versions: ['7.2', '7.3', '7.4']
+                composer-args: ['', '--prefer-lowest']
+                allow-failure: [false]
+                include:
+                    - operating-system: ubuntu-latest
+                      php-versions: '8.0'
+                      composer-args: '--ignore-platform-reqs --no-scripts'
+                      allow-failure: true
+                    - operating-system: ubuntu-latest
+                      php-versions: '8.0'
+                      composer-args: '--prefer-lowest --ignore-platform-reqs --no-scripts'
+                      allow-failure: true
+        runs-on: ${{ matrix.operating-system }}
+        continue-on-error: ${{ matrix.allow-failure }}
```

Comment:
This allows the PHP 8 build to fail, but the merge-me task to still run. Which isn't too important here, since PHP 8 is passing, but may be good if we want to have `nightly` tests, or have a lib that PHP 8 isn't passing but we're still working on a fix.